app/views/errors/validation_error.html.slim

```diff
@@ -1,3 +1,11 @@
-h1 The change you wanted was rejected.
-h2 Maybe you tried to change something you didn't have access to.
-p If you are the application owner check the logs for more information.
+.flex-container
+  .flex-item
+    .validation_error_logo
+  .flex-item
+    h2 The change you wanted was rejected.
+    hr/
```

Comment:
This is the old version. I'm currently working on 404, once I will make it working, I will change 422.